wgengine/magicsock/magicsock.go

```diff
@@ -3647,6 +3647,9 @@ type endpoint struct {
 	// See #540 for background.
 	heartbeatDisabled bool
 	pathFinderRunning bool
+
+	// whether the node has expired
```

Comment:
If it's a before-line comment, it should be a complete sentence "// expired is whether .."

But move it down to a same-line comment and this is fine